docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Ingesting Dynamo DB tables into the Landing Zone
+description: "Ingesting tables from a Dynamo DB instance into the Data Platform landing zone"
+layout: playbook_js
+tags: [playbook]
+---
+
+## Objective
+
+To setup a job that will ingest specified tables from a Dynamo DB instance.
+
+## Intended audience
+
+- Data Engineers
+- Data Analysts
+
+## Prerequisites
+
+- You have a Github account - you can [create one][github_signup] yourself using your Hackney email
```

Comment:
The first two points are already prerequisites for getting set up on the platform so not sure they're needed here